src/main/java/uk/gov/companieshouse/chd/order/api/validator/CreateItemRequestValidator.java

```diff
@@ -43,7 +44,7 @@
     }
 
     private void checkString(String string, List<String> errors, String errorMessage) {
-        if(string == null || string.isEmpty()){
+        if(StringUtils.isBlank(string)){
```

Comment:
If you're really keen on brevity you could use a static import for the `isBlank()` method.